extensions/sql-migration/src/dialog/validationResults/valideIRDialog.ts

```diff
@@ -0,0 +1,658 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as constants from '../../constants/strings';
+import { validateIrDatabaseMigrationSettings, validateIrSqlDatabaseMigrationSettings } from '../../api/azure';
+import { MigrationStateModel, MigrationTargetType, ValidateIrState, ValidationResult } from '../../models/stateMachine';
+import { EOL } from 'os';
+import { IconPathHelper } from '../../constants/iconPathHelper';
+
+const DialogName = 'ValidateIrDialog';
+
+enum Result {
```

Comment:
likewise this enum is a little confusing to me. Are these meant to refer to column indexes or something? Could be renamed for more clarity